packages/core/src/math/int512.rs

```diff
@@ -643,7 +645,9 @@ where
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::{from_json, math::conversion::test_try_from_uint_to_int, to_json_vec};
+    use crate::math::conversion::test_try_from_uint_to_int;
+
+    use cosmwasm_std::{from_json, to_json_vec};
```

Comment:
maybe we can use serde_json directly here to avoid cosmwasm_core depending on cosmwasm_std

Reply:
Will have a look. I *don't think* it has any dependencies on `std`-only code